Exception in Play 2.6.0-RC2

Library doesn't work since Scla Play 2.6.0-RC2. 

`[RuntimeException: java.lang.NoSuchMethodError: play.api.libs.json.JsArray.value()Lscala/collection/Seq;]`

full stack trace:
`play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[RuntimeException: java.lang.NoSuchMethodError: play.api.libs.json.JsArray.value()Lscala/collection/Seq;]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:257)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:184)
	at play.core.server.AkkaHttpServer$$anonfun$$nestedInanonfun$executeHandler$1$1.applyOrElse(AkkaHttpServer.scala:228)
	at play.core.server.AkkaHttpServer$$anonfun$$nestedInanonfun$executeHandler$1$1.applyOrElse(AkkaHttpServer.scala:227)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:412)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:37)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)
	at scala.concurrent.impl.Promise$KeptPromise$Kept.onComplete(Promise.scala:368)
Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodError: play.api.libs.json.JsArray.value()Lscala/collection/Seq;
	at play.api.mvc.ActionBuilder$$anon$2.apply(Action.scala:424)
	at play.api.mvc.Action.$anonfun$apply$2(Action.scala:96)
	at play.api.libs.streams.StrictAccumulator.$anonfun$mapFuture$4(Accumulator.scala:174)
	at scala.util.Try$.apply(Try.scala:209)
	at play.api.libs.streams.StrictAccumulator.$anonfun$mapFuture$3(Accumulator.scala:174)
	at scala.Function1.$anonfun$andThen$1(Function1.scala:52)
	at scala.Function1.$anonfun$andThen$1(Function1.scala:52)
	at scala.Function1.$anonfun$andThen$1(Function1.scala:52)
	at play.api.libs.streams.StrictAccumulator.run(Accumulator.scala:211)
	at play.core.server.AkkaHttpServer.executeAction(AkkaHttpServer.scala:275)
Caused by: java.lang.NoSuchMethodError: play.api.libs.json.JsArray.value()Lscala/collection/Seq;
	at com.eclipsesource.schema.internal.validators.ArrayValidator$.validate(ArrayValidator.scala:17)
	at com.eclipsesource.schema.internal.validators.ArrayValidator$.validate(ArrayValidator.scala:13)
	at com.eclipsesource.schema.package$SchemaTypeExtensionOps.validateConstraints(package.scala:137)
	at com.eclipsesource.schema.package$SchemaTypeExtensionOps.validate(package.scala:102)
	at com.eclipsesource.schema.internal.validators.ObjectValidator$.$anonfun$validateProps$3(ObjectValidator.scala:57)
	at scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
	at scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
	at scala.collection.immutable.List.foldLeft(List.scala:86)
	at com.eclipsesource.schema.internal.validators.ObjectValidator$.$anonfun$validateProps$1(ObjectValidator.scala:43)
	at scalaz.package$ReaderWriterState$.$anonfun$apply$3(package.scala:189)`

--- a/project/Build.scala
+++ b/project/Build.scala
@@ -4,8 +4,8 @@ import sbt._
 
 object Version {
   // TODO: remove milestone versions once stable builds are available
-  final val playJson      = "2.6.0-M5"
-  final val playTest      = "2.6.0-M2"
+  final val playJson      = "2.6.0-RC2"
+  final val playTest      = "2.6.0-RC2"
   final val scalaz        = "7.2.8"
   final val specs2        = "3.8.6"
   final val guava         = "19.0"

--- a/src/test/scala/com/eclipsesource/schema/SchemaValidatorSpec.scala
+++ b/src/test/scala/com/eclipsesource/schema/SchemaValidatorSpec.scala
@@ -158,7 +158,7 @@ class SchemaValidatorSpec extends PlaySpecification {
       )
       val result: JsResult[Foo] = SchemaValidator().validate(schema, fooInstance, lr)
       result.asEither must beLeft.like { case error =>
-        (error.toJson(0).get \ "instancePath") must beEqualTo(JsDefined(JsString("/loc")))
+        (error.toJson(0) \ "instancePath") must beEqualTo(JsDefined(JsString("/loc")))
       }
     }
 
@@ -380,7 +380,7 @@ class SchemaValidatorSpec extends PlaySpecification {
         )
       )
     val errors = result.asEither.left.get
-    val error = errors.toJson(0).get
+    val error = errors.toJson(0)
     error \ "errors" \ "/allOf/0" \ 0 \ "origin" must beEqualTo(JsDefined(JsString("#/allOf/0/properties/something")))
     error \ "errors" \ "/allOf/0" \ 0 \ "resolutionScope" must beEqualTo(JsDefined(JsString("common-schema.json")))
     error \ "errors" \ "/allOf/0" \ 0 \ "schemaPath" must beEqualTo(JsDefined(JsString("#/definitions/foo")))

--- a/src/test/scala/com/eclipsesource/schema/test/Assets.scala
+++ b/src/test/scala/com/eclipsesource/schema/test/Assets.scala
@@ -10,11 +10,14 @@ object Assets {
 
   def routes(clazz: Class[_], prefix: String = ""): PartialFunction[(String, String), Handler] = {
     case (_, path) =>
-
       try {
-        Action(Ok.sendResource(prefix + path.substring(1), clazz.getClassLoader))
+        val resourceName = prefix + path.substring(1)
+        Option(clazz.getClassLoader.getResource(resourceName))
+          .map(_ => Action(Ok.sendResource(resourceName, clazz.getClassLoader)))
+          .getOrElse(Action(BadRequest(s"$resourceName not found.")))
       } catch {
-        case ex: Throwable => Action(BadRequest(ex.getMessage))
+        case ex: Throwable =>
+          Action(BadRequest(ex.getMessage))
       }
   }
 }

